test/cpp.cc

```diff
@@ -6,9 +6,20 @@
  * Authors: Keith Busch <keith.busch@wdc.com>
  */
 
+#include <algorithm>
 #include <iostream>
 #include <libnvme.h>
 
+static int min_compile_test()
+{
+	return std::min(1, 2);
+}
+
+static int max_compile_test()
+{
+	return std::max(1, 2);
+}
```

Comment:
This is unrelated with the above change.

Reply:
yup, I think min and max should always appear in pairs, so I just added it, remove now.